drivers/amazonec2/stub_test.go

```diff
@@ -73,7 +77,9 @@ type fakeEC2WithLogin struct {
 	*fakeEC2
 }
 
-func (f *fakeEC2WithLogin) DescribeAccountAttributes(input *ec2.DescribeAccountAttributesInput) (*ec2.DescribeAccountAttributesOutput, error) {
+func (f *fakeEC2WithLogin) DescribeAccountAttributes(
```

Comment:
Yes, `gofmt` does allow this, ie: this is a no-diff `gofmt -s -w drivers/` - that said - 2 go noobs here who are used to wrapping at 80 or 100 cols by long convention in other langs.

Happy to undo this.